tests/async_client_test.py

```diff
@@ -749,6 +749,8 @@ async def test_sync(self, async_client, aioresponse):
         # Test with filter dict
 
         aioresponse.get(
+            # TODO: what the hell is the difference here?
```

Comment:
I think it's fine to just use the more specific URL.

Reply:
I'm hitting that issue as well on master while migrating our CI over, seems like something is double quoting, though it happens only when using Tox so :shrug:

Reply:
Okay, if you're happy using the specified string that works for me. My most recent commits changes that + removes the trailing slash check, and I fixed up the types casting.